.github/detect/excludes.txt

```diff
@@ -31,4 +31,5 @@ packages/sdk/test/unit/node/appCredential.spec.ts
 packages/sdk/test/unit/node/core/onBehalfOfUserCredential.spec.ts
 packages/fx-core/tests/core/samples_v2.zip
 packages/cli/tests/e2e/manifest/appPackage.dev.zip
-packages/vscode-extension/src/debug/teamsfxDebugProvider.ts
\ No newline at end of file
+packages/fx-core/templates/plugins/resource/cicd/azdo/*.yml
```

Comment:
why do we need to exclude these yaml file?

Reply:
Because the code snippet `NodeTool@0` is recognized as sensitive password in the check progress.